scripts.js

```diff
@@ -0,0 +1,226 @@
+    const disableBtn = this.slides[this.countSlide].querySelectorAll('.radio');
+    Array.from(disableBtn).forEach(item => item.setAttribute("disabled", "disabled"));
+
+    const disableAnswers = this.slides[this.countSlide].querySelector('.question__answers');
+    disableAnswers.removeEventListener("click", this.checkResultBind);
+  }
+
+  startGame() {
+    this.start.classList.add('none');
+    this.next.classList.remove('none');
+    this.previous.classList.remove('none');
+    this.switchQuestion(this.countSlide);
+    console.log(`start ${this.countSlide + 1}`);
+  }
+
+  switchQuestion(switchedSlide) {
+    this.slides[this.countSlide].classList.remove('active');
+    this.slides[switchedSlide].classList.add('active');
+
+    if (this.timerShowQuestion !== null) clearTimeout(this.timerShowQuestion);
+    if (this.countSlide !== 3) {
+      this.timerShowQuestion = setTimeout(() => {
+        this.disableButton();
+        this.switchNextQuestion();
+      }, 10000);
+    }
+  }
+
+  switchNextQuestion() {
+    if (this.countSlide === 2) {
+      this.next.textContent = 'check';
+    }
+
+    if (this.countSlide === 3) {
+      this.next.classList.add('none');
+      this.previous.classList.add('none');
+    }
+
+    if (this.countSlide === 4) return;
+
+
+    this.switchQuestion(this.countSlide + 1);
+    this.countSlide++;
+    console.log(`next ${this.countSlide + 1}`);
+    clearTimeout(this.timerCheckQuestion);
+  }
+
+  switchPreviousQuestion() {
+    if (this.countSlide <= 0) {
+      return;
+    }
+
+    if (this.countSlide != 2) {
+      this.next.textContent = 'next';
+    }
+
+    this.switchQuestion(this.countSlide - 1);
+    this.countSlide--;
+    console.log(this.countSlide + 1);
+  }
+
+  eventButtonName() {
+    let name = this.formName.value;
+    let err = document.createElement('p');
+    if (!this.regex.test(name)) {
+      err.classList.add('error');
+      err.textContent = 'Введите корректное имя(содержит 2-10 символов и начинается с заглавной буквы)';
+      this.form.append(err);
+    } else {
+      this.slides[4].classList.remove('active');
+      this.showResult(name);
+      this.restart.classList.remove('none');
+    }
+    setTimeout(() => err.remove(), 2000);
+    this.formName.value = null;
+  }
+
+  showResult(name) {
+    this.counterAnswers.classList.remove('none');
+    this.counterAnswers.innerHTML = `Число правильных ответов игрока ${name} ${this.countAnswers}`;
+  }
+
+  eventRestart() {
+    this.restart.classList.add('none');
+    this.counterAnswers.classList.add('none');
+    this.start.classList.remove('none');
+    this.countSlide = 0;
+    this.countAnswers = 0;
+    this.assignHandlerAnswer();
+    this.activeButton();
+    this.correctRequestAnswers = [];
+    this.allAnswersRequest = [];
+    this.questionRequest = [];
+    this.makeRequest();
+    this.preloader.classList.remove('none');
+  }
+
+  activeButton() {
+    const radio = this.container.querySelectorAll('.radio');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/voronkova-ab/game-js/pullRequest?prid=7705696)
```suggestion
    const radio = this.container.querySelectorAll(".radio");
```